cmd/osv-scanner/fix/main.go

```diff
@@ -89,12 +89,22 @@ func Command(stdout, stderr io.Writer, r *reporter.Reporter) *cli.Command {
 				Name:  "maven-registry",
 				Usage: "URL of the default Maven registry to fetch metadata",
 			},
-
 			&cli.BoolFlag{
 				Name:  "non-interactive",
-				Usage: "run in the non-interactive mode",
+				Usage: "[DEPRECATED] run in the non-interactive mode",
 				Value: !term.IsTerminal(int(os.Stdin.Fd())), // Default to non-interactive if not being run in a terminal
```

Comment:
This flag doesn't actually do anything now, does it?
Maybe make it `Hidden`, and set the `Value` to true.